fitbenchmarking/fitbenchmarking
# Add documentation page for support pages

#### Description of Work

Adds documentation to the support pages.

Fixes #839 

#### Testing Instructions

1. Proofread the docs
2. Verify if confusing or elements are still missing.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
